pkg/cloudprovider/providers/gce/gce.go

```diff
@@ -87,6 +87,11 @@ func init() {
 	cloudprovider.RegisterCloudProvider(ProviderName, func(config io.Reader) (cloudprovider.Interface, error) { return newGCECloud(config) })
 }
 
+// Raw access to the underlying GCE service, probably should only be used for e2e tests
```

Comment:
Suggest a warning comment indicating that anyone who wanted access to a particular resource should probably go through the interface because of the error prone nature of handling and waiting for cloud operations.